install_config/master_node_configuration.adoc

```diff
@@ -33,259 +33,698 @@ remember all of the options to specify. To make this easier, the configuration
+If no names are specified, the `*ServiceAccountsController*` will not be
+started.
+
+|`*MasterCA*`
+|The CA for verifying the TLS connection back to the master. The service account
+controller will automatically inject the contents of this file into pods so they
+can verify connections to the master.
+
+|`*PrivateKeyFile*`
+|A file containing a PEM-encoded private RSA key, used to sign service account
+tokens. If no private key is specified, the service account `*TokensController*`
+will not be started.
+
+|`*PublicKeyFiles*`
+|A list of files, each containing a PEM-encoded public RSA key. If any file
+contains a private key, the public portion of the key is used. The list of
+public keys is used to verify presented service account tokens. Each key is
+tried in order until the list is exhausted or verification succeeds. If no keys
+are specified, no service account authentication will be available.
+
+|`*ServiceAccountConfig*`
+|Holds the necessary configuration options for a service account.
+
+|===
+
+[[master-config-serving-information-config]]
+=== Serving Information Configuration
+
+.Serving Information Configuration Parameters
+[cols="3a,6a",options="header"]
+|===
+
+| Parameter Name | Description
+
+|`*AllowRecursiveQueries*`
+|allows the DNS server on the master to answer queries recursively. Note that
+open resolvers can be used for DNS amplification attacks and the master DNS
+should not be made accessible to public networks.
+
+|`*BindAddress*`
+|The *ip:port* to serve on.
+
+|`*BindNetwork*`
+|Controls limits and behavior for importing images.
+
+|`*CertFile*`
+|A file containing a PEM-encoded certificate.
+
+|`*CertInfo*`
+|TLS cert information for serving secure traffic.
+
+|`*ClientCA*`
+|The certificate bundle for all the signers that you recognize for incoming
+client certificates.
+
+|`*dnsConfig*`
+|holds the necessary configuration options for DNS.
+
+|`*DNSDomain*`
+|Holds the domain suffix.
+
+|`*DNSIP*`
+|Holds the IP.
+
+|`*KeyFile*`
+|A file containing a PEM-encoded private key for the certificate specified by
+`*CertFile*`.
+
+|`*MasterClientConnectionOverrides*`
+|Provides overrides to the client connection used to connect to the master.
+
+|`*MaxRequestsInFlight*`
+|The number of concurrent requests allowed to the server. If zero, no limit.
+
+|`*NamedCertificates*`
+|A list of certificates to use to secure requests to specific host names.
+
+|`*RequestTimeoutSecond*`
+|The number of seconds before requests are timed out. The default is 60 minutes.
+If -1, there is no limit on requests.
+
+|`*ServingInfo*`
+|The HTTP serving information for the assets.
+
+|===
+
+[[master-node-config-volume-config]]
+=== Volume Configuration
+
+.Volume Configuration Parameters
+[cols="3a,6a",options="header"]
+|===
+
+| Parameter Name | Description
+
+|`*DynamicProvisioningEnabled*`
+|A boolean that toggles dynamic provisioning off when false, defaults to true.
```

Comment:
I'm not sure the above is saying as intended. Maybe just:

"A boolean to enable or disable dynamic provisioning. Default is true."